src/middlewares/error.middleware.js

```diff
@@ -0,0 +1,18 @@
+const { ApiError } = require('../exceptions/api.error');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
+
```

Comment:
The function should return after sending a response for an ApiError to prevent executing the code below. Consider adding a return statement after line 9.